javascript/example_code/iam/iam_attachrolepolicy.js

```diff
@@ -24,27 +24,33 @@ var paramsRoleList = {
   RoleName: process.argv[2]
 };
 
-iam.listAttachedRolePolicies(paramsRoleList, function(err, data) {
+var policyName = 'AmazonDynamoDBFullAccess';
+var policyArn = 'arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess';
+
+iam.listAttachedRolePolicies(paramsRoleList).eachPage(function(err, data) {
```

Comment:
I paginated over the examples, because it is possible that the number of roles an account has exceeds the number returned by the service in a single request.